frontend/src/metabase/nav/components/search/SearchResultsDropdown/SearchResultsDropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { jt } from "ttag";
+import { SearchResults } from "metabase/nav/components/search/SearchResults";
+import type { SearchResults as SearchResultsType } from "metabase-types/api";
+import type { WrappedResult } from "metabase/search/types";
+import { Text } from "metabase/ui";
+import { Icon } from "metabase/core/components/Icon";
+import {
+  SearchDropdownFooter,
+  SearchResultsContainer,
+} from "./SearchResultsDropdown.styled";
+
+export type SearchResultsDropdownProps = {
+  searchText: string;
+  onSearchItemSelect: (item: WrappedResult) => void;
+  goToSearchApp: () => void;
+};
+
+export const SearchResultsDropdown = ({
+  searchText,
+  onSearchItemSelect,
+  goToSearchApp,
+}: SearchResultsDropdownProps) => {
+  const renderFooter = (metadata: Omit<SearchResultsType, "data">) =>
```

Comment:
If we export `SearchResultsProps`  we could use `SearchResultsProps["footer"]` to avoid typing it again here